src/Tests/Subscriptions/When_caching.cs

```diff
@@ -15,17 +15,17 @@ namespace NServiceBus.Persistence.AzureStorage.ComponentTests.Subscriptions
     public class When_caching
     {
         [SetUp]
-        public void Setup()
+        public Task Setup()
         {
-            SubscriptionTestHelper.PerformStorageCleanup();
+            return SubscriptionTestHelper.PerformStorageCleanup();
         }
 
         [Test]
         public async Task Cached_get_should_be_faster()
         {
-            var persister = SubscriptionTestHelper.CreateAzureSubscriptionStorage();
+            var persister = await SubscriptionTestHelper.CreateAzureSubscriptionStorage();
             var type = new MessageType("type1", new Version(0, 0, 0, 0));
-            persister.Subscribe(new Subscriber("address://test-queue", "endpoint"), type, null).Await();
+            persister.Subscribe(new Subscriber("address://test-queue", "endpoint"), type, null).AwaitFix();
```

Comment:
Can't we just make these tests properly awaiting instead of using this extension method?

Reply:
@danielmarbach no, they fail.